### PR TITLE
S1 task 1

### PR DESCRIPTION
**Pull request by damienscarpellini**
please check code in functions page and showindivudual page.

it will now cycle through the entire collection and will recycle to the front of back if you go beyond the scope of the database.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+session_start();
```
Comment (waseembrare) [inline functions.php:2]: nice touch, including session in there

File: index.php [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+    $query->execute();
+    $allResults = $query->fetchAll();
+?>
 
 <!DOCTYPE html>
     <html lang="en">
     <head>
         <link rel="stylesheet" type="text/css" href="stylesheet.css">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta charset="UTF-8">
```
Comment (waseembrare) [inline index.php:8]: keep your code very neat, like it

File: showindividual.php [hunk 3]
```diff
@@ -1,23 +1,79 @@
-<?php ?>
+<?php
+    require_once('functions.php');
+    $db = new PDO('mysql:host=db; dbname=collections-project2', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $allItems = $db->prepare("SELECT * FROM `collections`;");
+    $allItems->execute();
+    $allResults = $allItems->fetchAll();
+    $numberOfItemInArray = count($allResults);
+    $lastItem = $allResults[$numberOfItemInArray - 1];
```
Comment (waseembrare) [inline showindividual.php:9]: Have no idea, but likely that there is a predefined function for this in php. If not then clever work around.

**Review by waseembrare — APPROVED** [review]
- Looks really good mate, really clean and easy to read code. I know its what we should all be doing but your code just reads really well (from my perspective anyway - lets hope Rayna thinks so too (-: !)

File: showindividual.php (hunk 3 above)
Comment (damienscarpellini) [reply showindividual.php:9]: function in functions page ahaha

**Pull request merged**